change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If I open the select by tabbing to it with the keyboard and pressing Enter, the filter input doesn't have focus. And I can't see any gesture to give it focus. Is that intentional?

Reply:
Nope, that's just a goof...I'll fix that.